src/server/live-reload/live-reload-server.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var fs = require('fs');
+var path = require('path');
+var telemetry = require('../telemetry-helper');
+var Watcher = require('./watcher').Watcher;
+
+var socket;
+var watcher;
+
+function cssFileChanged(fileRelativePath) {
+    telemetry.sendTelemetry('live-reload', { fileType: '.css' });
+    socket.emit('lr-update-css', { href: fileRelativePath.replace(/\\/g, '/') });
+}
+
+function nonCssFileChanged(fileRelativePath) {
+    telemetry.sendTelemetry('live-reload', { fileType: path.extname(fileRelativePath) });
+    socket.emit('lr-full-reload');
+}
+
+module.exports.init = function (sock) {
+    if (!watcher) {
+        watcher = new Watcher(cssFileChanged, nonCssFileChanged);
+        watcher.startWatching();
+    }
+
+    socket = sock;
```

Comment:
Will this work for multiple app browsers at a time? We don't need to support it for right away, but @busykai mentioned he will be looking into supporting that, so it is good to keep that scenario in mind.